src/core/server/base.js

```diff
@@ -312,6 +311,7 @@ export function runServer({ listen = true, app = appName } = {}) {
     })
     .catch((err) => {
       log.error({ err });
+      throw err;
```

Comment:
I would have thought the logging should be enough. Another potentially relevant issue is the one for adding the sentry client which should help for tracking down caught and uncaught server exceptions. https://github.com/mozilla/addons-frontend/issues/1559